pandas/tests/io/parser/test_common.py

```diff
@@ -969,6 +969,18 @@ def test_nonexistent_path(all_parsers):
         assert path == filename
 
 
+def test_no_permission(all_parsers):
+    # GH 23784
+    parser = all_parsers
+
+    msg = r"\[Errno 13\]"
+    with tm.ensure_clean() as path:
+        os.chmod(path, 0)  # make file unreadable
+        with pytest.raises(PermissionError, match=msg) as e:
+            parser.read_csv(path)
+            assert path == e.value.filename
```

Comment:
i think this assertion needs to be un-indented

Reply:
You are absolutely right and the test before this one had the same issue (I copied it from that test). That one is fixed as well.